### DISCUSSION TITLE
Ender 3 V2 Neo alignment is off after pausing a print.

### DISCUSSION THREAD

**Discussion by datsunpickup** [mriscoc/Ender3V2S1#1239] [General]
I am having a issue where my alignment is off after restarting from either a machine or gcode pause.  Any suggestions?

Thanks

**Comment by mriscoc** [comment]
Check your parking position

**Comment by datsunpickup** [comment]
Thank you

Sent from Samsung Galaxy smartphone.
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
